File: estudo/neri/algoritmo/c/Venda.c
#include <stdio.h>
int main(){
    float salario, venda, comissao;
    salario = 800;
    printf("Informe o valor da venda:");
    scanf("%f",&venda);
    comissao = (15 * venda) /100;
    salario = salario + comissao;
    printf("A venda foi de: %f",venda);
    printf("A comissao foi de: %f",comissao);
    printf("O valor do salario foi de: %f",salario);
    
}

File: estudo/neri/algoritmo/java/Venda.java
import javax.swing.JOptionPane;

public class Venda {
    public static void main(String args[]){
        float salario, venda, comissao;
        salario = 800;
        venda = Float.parseFloat(JOptionPane.showInputDialog(null,"Informe o valor da venda"));
        comissao = (15 * venda) /100;
        salario = salario + comissao;
        JOptionPane.showMessageDialog(null,"A comissao foi de: "+comissao);
        JOptionPane.showMessageDialog(null,"O salario total foi de: "+salario);
    }
}
